playbook/app/pb_kits/playbook/pb_button/_button.tsx

```diff
@@ -116,11 +116,19 @@ const Button = (props: ButtonPropTypes): React.ReactElement => {
   const content = (
     <span className="pb_button_content">
       {icon && !iconRight && (
-        <i className={`pb_icon_kit far fa-${icon} fa-fw button_with_icon`} />
+        <i className={`pb_icon_kit button_with_icon`}>
```

Comment:
@skduncan , we shouldn't need to add an extra <i> around our <Icon /> kit. 
I think we can just added `button_with_icon` to the className of the Icon instance... like this:

```suggestion
        <Icon className='button_with_icon'>
```

Any reason we needed to wrap that <Icon /> with an <i> element?